### PR TITLE
[CELEBORN-689] Fix the incorrect part of PushDataHandler message type converted to status code

### PR DESCRIPTION
**Pull request by cxzl25**

### CODE REVIEW

File: worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala [hunk 1]
```diff
@@ -991,7 +991,7 @@ class PushDataHandler extends BaseMessageHandler with Logging {
 
     val (messageMaster, messageSlave) =
       messageType match {
-        case Type.PUSH_DATA | Type.PUSH_DATA_HAND_SHAKE =>
```
Comment (cxzl25) [inline worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala:994]: duplicate match

https://github.com/apache/incubator-celeborn/blob/7d7107d607b853ab61b8299f83de2e3dacfa7e3c/worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala#L994-L1000

**Review by pan3793 — APPROVED** [review]

**Comment by RexXiong** [comment]
@zhongqiangczq pls take a look, if we need modify shuffleClient at the same time.

File: worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala (hunk 1 above)
Comment (zhongqiangczq) [reply worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala:994]: ok, the duplicate match can be removed

**Comment by zhongqiangczq** [comment]
Now no need to care about this. 
the message PUSH_DATA_HAND_SHAKE  is used by flinkShuffleClient. Currently, flinkShuffleClient don't process this statuscode.

**Review by zhongqiangczq — APPROVED** [review]
LGTM

**Comment by pan3793** [comment]
Thanks, merged to main/0.3

**Pull request closed without merging**